### ISSUE TITLE
Owner (character-rights = 3) Spawning Items crashes client

### ISSUE THREAD

**Issue by Sch4ff3r** [dginovker/2006rebotted#261]
Spawns item but also crashes client. When logging back in, the item is shown.

**Comment by DPSCryptic** [comment]
what do you mean with this?

**Comment by Sch4ff3r** [comment]
If my rights are set as 3.
When I ::item 1 1 example, when hitting enter, it crashes client.

**Comment by dginovker** [comment]
Setting your rights to 2 works. I have no idea what 3 is.

**Comment by DPSCryptic** [comment]
3 = owner

**Comment by Gptaqbc** [comment]
Item ID: 1 is the toolkit. No need to spawn that. Try 995, it's coins. If that doesnt work, there is a problem somewhere :P

**Comment by dginovker** [comment]
I can't reproduce this on my localhost:

![image](https://user-images.githubusercontent.com/32943174/70594257-e98f5b80-1bad-11ea-918a-70f123a4018e.png)

Running Java 1.8 by OpenJDK on Linux

**Comment by Sch4ff3r** [comment]
Running Windows-64 bit still crashes.

**Comment by RedSparr0w** [comment]
Cannot reproduce locally either.

`Java 1.8`
`Windows 10 64 bit`

**Comment by Gptaqbc** [comment]
Windows 7 Java 1.8. I never encountered any issue :/

**Comment by dginovker** [comment]
@Sch4ff3r open a command prompt and run `java --version` - what's the exact version it states?

**Comment by dginovker** [comment]
I actually was able to reproduce this issue again, after #293 as well. 

Spawned item `8839` on a character-rights = 2 and got this stacktrace in the client:


```
 Exception in thread "Thread-1" java.lang.ArrayIndexOutOfBoundsException: 8839
        at ItemDef.forID(ItemDef.java:193)
        at ItemDef.getSprite(ItemDef.java:2062)
        at Game.drawInterface(Game.java:7861)
        at Game.drawTabArea(Game.java:1726)
        at Game.drawGameScreen(Game.java:7559)
        at Game.processDrawing(Game.java:8402)
        at RSApplet.run(RSApplet.java:132)
        at Game.run(Game.java:4298)
        at java.lang.Thread.run(Thread.java:748)


```

**Comment by RedSparr0w** [comment]
That’s a separate issue, the highest item ID is 7955.
If you spawn anything above that, it won’t exist.
